cmd/createdkg.go

```diff
@@ -90,7 +92,11 @@ func runCreateDKG(ctx context.Context, conf createDKGConfig) (err error) {
 	}()
 
 	var operators []cluster.Operator
-	for _, opENR := range conf.OperatorENRs {
+	for i, opENR := range conf.OperatorENRs {
+		_, err := p2p.DecodeENR(opENR)
+		if err != nil {
+			return errors.Wrap(err, fmt.Sprintf("invalid ENR of operator %d", i))
```

Comment:
you should just be able to use `z.Int("index",i))` instead of `fmt.Sprintf`

Reply:
done